activities/member-relations/index.md

```diff
@@ -8,12 +8,13 @@ This outlines the stages by which a public organization becomes a founding membe
 
 ## 1. Expression of interest
 
-At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md).
+At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md). This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
 
-This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
+This stage may include the '[Information about Founding Membership](information-about-founding-membership.md)'.
```

Comment:
This is fine. But as for good form this link should not be in quotes and can be more descriptive.

Reply:
see #329